sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -42,8 +42,10 @@ add_executable (
     logging.cpp
     main.cpp
     nullable.cpp
+    operation.cpp
+    operation_status.cpp
     pipeline.cpp
-    policy.cpp
+    policy.cpp    
```

Comment:
The extra spacing issue still exists:
![image](https://user-images.githubusercontent.com/6527137/102539681-dd609880-4062-11eb-8c02-0a4af94f323c.png)